peps/pep-0725.rst

```diff
@@ -169,6 +173,8 @@ necessary to run a host dependency under an emulator, or through a custom tool
 like crossenv_. When host dependencies imply a runtime dependency, that runtime
 dependency also does not have to be declared, just like for build dependencies.
 
+.. JRG: Why? are host deps it added implicitly as runtime ones?
```

Comment:
Ah, you caught confused terminology. "host dependencies" is defined higher up as _"build-time dependencies built for the host machine"_. That could use some cleaning up indeed.